scripts/sipify.pl

```diff
@@ -133,8 +133,17 @@ sub processDoxygenLine {
     if ( $line =~ m/\\since .*?([\d\.]+)/i ) {
         return ".. versionadded:: $1\n";
     }
-    if ( $line =~ m/\\see (.*)/ ) {
-        return ".. seealso:: $1\n";
+    if ( $line =~ m/\\see *(.*)/ ) {
+        my $seealso = $1;
+        if (  $1 =~ m/^Qgs[A-Z]/ ) {
```

Comment:
i think this one also should check for `.`s, and if so use the func type link

Reply:
indeed